tests/handlers/protocols-query.spec.ts

```diff
@@ -117,23 +120,45 @@ export function testProtocolsQueryHandler(): void {
         });
 
         const conditionalQueryReply = await dwn.processMessage(alice.did, conditionalQuery.message);
-
         expect(conditionalQueryReply.status.code).to.equal(200);
         expect(conditionalQueryReply.entries?.length).to.equal(1); // only 1 entry should match the query on protocol
 
         const protocolConfigured = conditionalQueryReply.entries![0] as ProtocolsConfigureMessage;
         expect(protocolConfigured).to.deep.equal(protocol2.message);
 
-        // testing fetch-all query without filter
+        // testing authenticated but unauthorized (to private ProtocolsConfigures) conditional query
```

Comment:
The "to private ProtocolsConfigures" part of the comment was confusing a bit for me, had to re-read it a couple of times before I understood what it meant.
Maybe something like this is clearer?

```suggestion
        // testing authenticated but unauthorized conditional query. It should return only public messages.
```

Reply:
Fixed!